frontend/src/components/custom-tools/combined-output/JsonView.jsx

```diff
@@ -1,18 +1,38 @@
 import PropTypes from "prop-types";
 import Prism from "prismjs";
 import { useEffect } from "react";
+import { ProfileInfoBar } from "../profile-info-bar/ProfileInfoBar";
+import TabPane from "antd/es/tabs/TabPane";
+import { Tabs } from "antd";
 
-function JsonView({ combinedOutput }) {
+function JsonView({
+  combinedOutput,
+  handleTabChange,
+  adapterData,
+  activeKey,
+  selectedProfile,
+  llmProfiles,
+}) {
   useEffect(() => {
     Prism.highlightAll();
   }, [combinedOutput]);
 
   return (
     <div className="combined-op-layout">
       <div className="combined-op-header">
+        <Tabs activeKey={activeKey} onChange={handleTabChange} moreIcon={<></>}>
+          <TabPane tab={<span>{"Default"}</span>} key={"0"}></TabPane>
```

Comment:
@jagadeeswaran-zipstack NIT: We don't need curly brackets when render text inside a tag. 

```suggestion
          <TabPane tab={<span>Default</span>} key={"0"}></TabPane>
```